<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulation</class>
 <widget class="QWidget" name="Simulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>481</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>策略1</string>
    </attribute>
    <widget class="QLabel" name="method1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>策略2</string>
    </attribute>
    <widget class="QLabel" name="method2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>策略3</string>
    </attribute>
    <widget class="QLabel" name="method3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>策略4</string>
    </attribute>
    <widget class="QLabel" name="method4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Begin_Simulation">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>430</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始模拟</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Return">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>490</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>481</width>
     <height>251</height>
    </rect>
   </property>
   <widget class="QSpinBox" name="Rt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rt指数大小：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0代病人数量：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Zero">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>模拟总人数：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="People">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3000</number>
    </property>
    <property name="maximum">
     <number>8000</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="GeLi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>隔离速度</string>
    </property>
    <widget class="QRadioButton" name="Slow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>112</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>较慢</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Middle">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>112</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>一般</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Fast">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>112</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>较快</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="YiMiao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>疫苗接种速度</string>
    </property>
    <widget class="QRadioButton" name="Slow_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>112</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>较慢</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Middle_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>112</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>一般</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Fast_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>112</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>较快</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Method">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>111</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>选择策略</string>
   </property>
   <widget class="QRadioButton" name="Method_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>策略1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Method_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>策略2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Method_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>策略3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Method_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>策略4</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
